lib/robots/dor_repo/gis_assembly/package_data.rb

```diff
@@ -66,12 +66,11 @@ def generate_data_zip(rootdir)
             end
           end
 
-          zipfn = File.join(rootdir, 'content', 'data.zip')
-          FileUtils.mkdir_p(File.dirname(zipfn)) unless File.directory?(File.dirname(zipfn))
-          FileUtils.rm_f(zipfn) if File.size?(zipfn)
+          FileUtils.mkdir_p(File.dirname(data_zip_filepath)) unless File.directory?(File.dirname(data_zip_filepath))
+          FileUtils.rm_f(data_zip_filepath) if File.size?(data_zip_filepath)
 
-          logger.debug "Compressing #{bare_druid} into #{zipfn}"
-          system("zip -v#{recurse_flag ? 'r' : ''} '#{zipfn}' #{fns.join(' ')}")
+          logger.debug "Compressing #{bare_druid} into #{data_zip_filepath}"
+          system("zip -v#{recurse_flag ? 'r' : ''} '#{data_zip_filepath}' #{fns.join(' ')}")
         end
```

Comment:
`zipfn` --> `data_zipe_filepath` for consistency (and pass in this value from the caller)